sickchill/providers/metadata/musicbrainz.py

```
@@ -0,0 +1,720 @@
+"""MusicBrainz metadata provider for SickChill."""
+
+import logging
+import os
+from typing import Dict, List, Optional, Union
+from xml.etree import ElementTree
+
+import musicbrainzngs
+import requests
+
+from sickchill import logger, settings
+from sickchill.oldbeard.databases import music
```

Ruff (0.12.2)

`sickchill.oldbeard.databases.music` imported but unused

Remove unused import: `sickchill.oldbeard.databases.music`

(F401)